specification/hybridconnectivity/resource-manager/Microsoft.HybridConnectivity/PublicCloud/stable/2024-12-01/publicCloud.json

```diff
@@ -1,1680 +0,0 @@
-{
-  "swagger": "2.0",
```

Comment:
autorest generation failed with this swagger is removed, should we update the readme also?

Reply:
Yes, please.